src/images/svg/terms/bug-bounty.svg

```diff
@@ -0,0 +1,12 @@
+<svg width="232" height="145" viewBox="0 0 232 145" xmlns="http://www.w3.org/2000/svg">
+    <defs>
+        <linearGradient x1="0%" y1="100%" x2="100%" y2="0%" id="a">
+            <stop stop-color="#FF444F" offset="0%"/>
```

Comment:
We usually minify our SVGs running `svgo` followed by the path of the file/directory. You may need to install it globally before if you haven't already.

Reply:
Tnx for your revision. I installed it globally. what else should I do in my local?